### DISCUSSION TITLE
启动图设计

### DISCUSSION THREAD

**Discussion by zty012** [graphif/project-graph#108] [通用]
启动图即应用加载时显示的画面

**Comment by zty012** [comment]
![image](https://github.com/user-attachments/assets/1d3ec8c9-1b2a-4eaa-8af8-0c7a8dcfecf7)

**Comment by Littlefean** [comment]
![project-graph-start](https://github.com/user-attachments/assets/0cf9bd78-5cae-4eb3-9bcc-843ea0838452)
